docs/src/starknet/account.md

```diff
@@ -102,15 +103,15 @@ For a detailed CLI description, see [account delete command reference](../append
 
 ### Custom Account Contract
 
-By default, `sncast` creates/deploys an account using [openzeppelin contract's class hash](https://starkscan.co/class/0x058d97f7d76e78f44905cc30cb65b91ea49a4b908a76703c54197bca90f81773).
+By default, `sncast` creates/deploys an account using [OpenZeppelin's contract class hash](https://starkscan.co/class/0x00e2eb8f5672af4e6a4e8a8f1b44989685e668489b0a25437733756c5a34a1d6).
 It is possible to create an account using custom openzeppelin contract declared to starknet. This can be achieved
```

Comment:
btw: Doesn't this work with other accounts?

Reply:
Yes, updated that